app/services/suggester/users/recent.rb

```diff
@@ -55,15 +55,21 @@ def recent_commenters(num_comments = 2, limit = 8)
       end
 
       def established_user_article_count
-        Rails.env.production? ? 4 : -1
+        Rails.cache.fetch("established_user_article_count", expires_in: 1.day) do
```

Comment:
Not married to the 1 day expiration, it could probably be longer. If anyone has an opinion, let me know!